andela/travel_tool_front
# #160195763 Allow users create comments

#### What does this PR do?
Allows users post and see comments

#### Description of Task to be completed?
- create actions, reducers and sagas for creating comments
- create `handleEditorChange` method to handle change events
- create handleSubmit methods to handle comment submission
- write tests for actions, reducers, components and sagas

#### How should this be manually tested?
- Clone the repo - `git clone https://github.com/andela/travel_tool_front.git`
- Change into repo directory - `cd travel_tool_front`
- Check out this branch - `git checkout ft-consume-comments-api-160195763`
- Install dependencies and run the app - `make start`
- Navigate to `localhost:3000` in a browser
- Login and navigate to the requests page
- Create a request if you don't have any
- Click on a request
- Post a comment
- You should see the comment in the comments thread

#### Any background context you want to provide?
You need to have your backend server running for you to test this feature. See the setup [here](https://github.com/andela/travel_tool_back/pull/29)

#### What are the relevant pivotal tracker stories?
[#160195763](https://www.pivotaltracker.com/story/show/160195763)

#### Screenshots (if appropriate)
![image](https://user-images.githubusercontent.com/25783086/45149064-9f584700-b1c0-11e8-8659-1e4cc8a38978.png)


#### Questions:
None
